Dockerfile

```diff
@@ -7,4 +7,8 @@ COPY operator /bin/operator
 # On busybox 'nobody' has uid `65534'
 USER 65534
 
+LABEL org.opencontainers.image.source="https://github.com/prometheus-operator/prometheus-operator" \
```

Comment:
Maybe we should add `org.opencontainers.image.url` too?

Reply:
Added :+1: